assets/src/edit-story/elements/text/edit.js

```diff
@@ -74,6 +74,7 @@ const TextBox = styled.div`
 	${elementWithBackgroundColor}
 	${elementWithFontColor}
 
+  opacity: ${({ opacity }) => (opacity ? opacity / 100 : null)};
```

Comment:
There's also a default `whiteout` color added to the edit mode for making it more visible, not sure if that should stay or not:
https://github.com/google/web-stories-wp/blob/03494a9f4ea4966eda21b251f1a71c1a7fee9cb8/assets/src/edit-story/components/canvas/editElement.js#L35-L42

Reply:
I'm ok to drop for now.